suite-native/storage/src/migrations/account/v3.ts

```diff
@@ -0,0 +1,23 @@
+import { Account, AccountType } from '@suite-common/wallet-types';
+import { getXpubOrDescriptorInfo } from '@trezor/utxo-lib';
```

Comment:
Just a thought: If user doesn't update for a year and this function changes, then updates the app and migration gets triggered, it could potentially cause some issues with the migration or crash the app.

Reply:
Good catch. After reducing the logic of the `getXpubOrDescriptorInfo` function of `@trezor/utxo-lib` and putting it directly to this file, there is no space for such an error. Fixed in 607860e903.

Reply:
Same could happen with types, it's better to inline "subset" of types into migration than importing them if possible, because someone may change Account type in future, which can cause some error here, but you should never change migration once it's out.

Reply:
I removed the Account and AccountType types in fixup 7695e9d8bd. But I left there `import { getNetworkType, NetworkSymbol } from '@suite-common/wallet-config';`, because without this line I would basically had to copy paste huge part of this file into the migration. Is it ok for you like it is?

Reply:
`getNetworkType` is a one-liner and `networkSymbol` can be defined in the migration file like this: 
```
type NetworkSymbol = "btc" | "ltc" | "eth" | "etc" | "xrp" | "bch" | "btg" | "dash" | "dgb" | "doge" | "nmc" | "vtc" | "zec" | "test" | "regtest" | "tsep" | "tgor" | "thol" | "txrp" | "ada" | "tada" | "sol" | "dsol"
```

Correct me if I am missing something.

Reply:
Generally, it would be perfect not to have any imports here. I understand this cannot always be done without huge amount of code so no need to be too strict about it. Finding a nice compromise here is what I'm trying to do. 

If you can paste in `getNetworkType` and `NetworkSymbol`, that would be fine with me, the `decode` function would need the whole file pasted in which might not be necessary, but let me know what you think.

Reply:
I was thinking that I would have to copy paste not only `type NetworkSymbol`, but also map which is mapping `NetworkSymbol` to `NetworkType` (functionality of `getNetworkType`). But I just realized, that networkType is already stored in the account, so the information is already present and the call of `getNetworkType` is redundant. Types added  according to your suggestion in 4606cc5d45, so there is no more suite file imports present.

On the other hand, copying the whole content of the `bs58.decode` is overkill IMO. If there would be any breaking change in this library, we would detect it before pushing it to the production.

Reply:
Yes, I think it's good like this. I'm approving, but maybe with this PR, I would ask @Nodonisko and @matejkriz to also review :)